Fix bug with not closing editorState after deleting the element in form mode (#1717)
finos/legend-studio: .changeset/fair-carrots-try.md

---
'@finos/legend-application-studio': patch
---

> Fix bug with not closing editorState after deleting the element in form mode.

Reviewer: ```suggestion
Fix with `Studio` not closing editor tabs for deleted elements
```
